### ISSUE TITLE
Fails on MacOS

### ISSUE THREAD

**Issue by itsMapleLeaf** [itsMapleLeaf/remix-tailwind#1]
CI run for reference: https://github.com/itsMapleLeaf/remix-tailwind/runs/4310332190?check_suite_focus=true#step:6:22

PostCSS has a dependency on fsevents, a mac-only native module, and Remix isn't configured to load `.node` files through esbuild, so we have a mac-only error.

**Comment by sync** [comment]
I tried patching remix.run dev module with this and it worked: https://github.com/evanw/esbuild/issues/1051

**Comment by itsMapleLeaf** [comment]
It sounds like a PR needs to be made to remix, then?

**Comment by sync** [comment]
[@remix-run+dev+1.0.6.patch.zip](https://github.com/itsMapleLeaf/remix-tailwind/files/7610910/%40remix-run%2Bdev%2B1.0.6.patch.zip)

**Comment by sync** [comment]
yes by the look of it, but not sure if it will fly

**Comment by sync** [comment]
Want me to do it ? (not sure if I will have time to do it this week)

**Comment by itsMapleLeaf** [comment]
> Want me to do it ? (not sure if I will have time to do it this week)

Yeah, since you already have the working and tested code, go for it :+1:

**Comment by sync** [comment]
I've sent a PR to remix, need to figure out how to test it
